packages/sit-onyx/src/components/OnyxAccordionItem/OnyxAccordionItem.vue

```diff
@@ -0,0 +1,168 @@
+  disabled: false,
+  skeleton: SKELETON_INJECTED_SYMBOL,
+});
+
+defineSlots<{
+  /**
+   * Displays the header content.
+   */
+  header(): unknown;
+
+  /**
+   * Displays the panel content.
+   */
+  default?(): unknown;
+}>();
+const itemId = useId();
+const accordionContext = inject(ACCORDION_INJECTION_KEY);
+
+const isOpen = computed({
+  get: () => accordionContext?.openItems.value.has(itemId) || props.open || false,
+  set: (value: boolean) => {
+    accordionContext?.updateOpen(itemId, value);
+  },
+});
+
+const skeleton = useSkeletonContext(props);
+const { densityClass } = useDensity(props);
+const isDiabled = computed(() => accordionContext?.disabled.value || props.disabled);
+</script>
+
+<template>
+  <div
+    v-if="skeleton || accordionContext?.skeleton.value"
+    :class="['onyx-component', 'onyx-accordion-item-skeleton', densityClass]"
+  >
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__main" />
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__icon" />
+  </div>
+
+  <details
+    v-else
+    class="onyx-component onyx-accordion-item"
+    :class="[densityClass]"
+    :open="isOpen"
+    @toggle="isOpen = ($event.target as HTMLDetailsElement).open"
+  >
+    <summary
+      :id="'header-' + itemId"
+      class="onyx-accordion-item__header"
+      role="button"
+      :tabindex="isDiabled ? -1 : 0"
+      :aria-expanded="isOpen"
+      :aria-controls="'panel-' + itemId"
+      :aria-disabled="isDiabled"
+    >
+      <div class="onyx-accordion-item__header-content">
+        <slot name="header"></slot>
+      </div>
+
+      <OnyxIcon :icon="chevronLeftSmall" class="onyx-accordion-item__header-icon" />
+    </summary>
+    <div
+      :id="'panel-' + itemId"
+      class="onyx-accordion-item__panel"
+      role="region"
+      :aria-labelledby="'header-' + itemId"
+    >
+      <slot></slot>
+    </div>
+  </details>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers";
+.onyx-accordion-item {
+  @include layers.component() {
+    border-bottom: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral);
+    color: var(--onyx-color-text-icons-neutral-intense);
+    font-family: var(--onyx-font-family);
+    position: relative;
+    width: 100%;
+
+    &__header {
+      width: 100%;
+      position: relative;
+      display: flex;
+
+      justify-content: space-between;
+      gap: var(--onyx-density-md);
+      align-items: center;
+      padding: var(--onyx-density-md);
+
+      cursor: pointer;
+      &:hover,
+      &:focus-visible {
+        background-color: var(--onyx-color-base-neutral-200);
+        outline: none;
+      }
+      &-content {
+        display: flex;
+        * {
+          font-size: 1rem;
+          font-weight: 400;
+        }
+      }
+
+      &-icon {
+        color: inherit;
+      }
+    }
+    &[open] &__header-icon {
+      transform: rotate(-90deg);
+    }
+    &:has(:focus-visible) {
```

Comment:
```suggestion
    &:has(&__header:focus-visible) {
```

Otherwise it would apply to any focus-visible elements, e.g. inputs inside the panel which is not intended I guess